### ISSUE TITLE
MasterWorker ExecuteOnceTask

### ISSUE THREAD

**Issue by activeeon-bot** [ow2-proactive/programming#374]
<a href="https://jira.activeeon.com/browse/PROACTIVE-493" title="PROACTIVE-493">Original issue</a> created by <a href="mailto:fabien.viale_AT_activeeon.com">Fabien Viale</a> on 14, Oct 2008 at 17:25 PM - PROACTIVE-493

<hr />

It should be possible to submit specific tasks in the MasterWorker which must be executed once on every worker and only once

**Comment by activeeon-bot** [comment]
<a href="https://jira.activeeon.com/browse/PROACTIVE-493?focusedCommentId=11255">Original comment</a> posted by <a href="mailto:clement.mathieu_AT_activeeon.com">Clement Mathieu</a> on 03, Mar 2009 at 15:56 PM

<hr />

What is the current status of this issue ? 4.1.0 should be released mid-march.

**Comment by activeeon-bot** [comment]
<a href="https://jira.activeeon.com/browse/PROACTIVE-493?focusedCommentId=11256">Original comment</a> posted by <a href="mailto:fabien.viale_AT_activeeon.com">Fabien Viale</a> on 03, Mar 2009 at 17:49 PM

<hr />

I postpone it for 4.2.0

**Comment by activeeon-bot** [comment]
<a href="https://jira.activeeon.com/browse/PROACTIVE-493?focusedCommentId=18583">Original comment</a> posted by <a href="mailto:youri.bonnaffe_AT_activeeon.com">Youri Bonnaffe</a> on 12, Dec 2014 at 12:54 PM

<hr />

Marking old issues as won't fix, either because we don't support it anymore, or it is needed or it has been there for too long.

Feel free to reopen if needed.
